src/test/java/com/google/sps/UserUtilTest.java

```diff
@@ -38,6 +38,7 @@
   // helper variable allows the use of entities in testing 
   private final LocalServiceTestHelper helper =
     new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  static final String sessionIdCookieName = "SessionID";
```

Comment:
Oh, you should use the `sessionIdCookieName` defined in UserUtil.java, rather than define a new one